src/spyglass/lfp/lfp_merge.py

```diff
@@ -0,0 +1,53 @@
+import datajoint as dj
+from spyglass.lfp.v1 import LFP, ImportedLFP
+
+schema = dj.schema("lfp")
+
+
+@schema
+class LFPOutput(dj.Manual):
+    definition = """
+    lfp_id: uuid
+    ---
+    stream: varchar(40)
+    """
+
+    class LFP(dj.Part):
+        definition = """
+        -> LFPOutput
+        ---
+        -> LFP
```

Comment:
The upstream LFP table should be called `LFPv1` if we're aiming to maintain consistency. See [here](https://github.com/LorenFrankLab/spyglass/blob/011dcaaca1f1dc20f1cc75c0edd8f2a85ad48ce4/src/spyglass/position/v1/position_dlc_selection.py#L35) for DLCPosV1